.golangci.yml

```diff
@@ -107,6 +107,21 @@ issues:
     - path: mongo_scaler.go
       linters:
         - dupl
+  # Exclude gci check for //+kubebuilder:scaffold:imports comments. Waiting to
+  # resolve https://github.com/kedacore/keda/issues/4379
+    - path: cmd/operator/main.go
+      linters:
+        - gci
+    - path: cmd/webhooks/main.go
+      linters:
+        - gci
+    - path: controllers/keda/suite_test.go
+      linters:
+        - gci
+    - path: apis/keda/v1alpha1/scaledobject_webhook_test.go
+      linters:
+        - gci
+
```

Comment:
Why is this required now?

Reply:
the gci check wants to remove the comments - i tried to move them around but doesnt work, im tracking the issue within gci here https://github.com/kedacore/keda/issues/4379